src/librustc_trait_selection/traits/mod.rs

```diff
@@ -422,11 +422,11 @@ where
 /// environment. If this returns false, then either normalize
 /// encountered an error or one of the predicates did not hold. Used
 /// when creating vtables to check for unsatisfiable methods.
-pub fn normalize_and_test_predicates<'tcx>(
+pub fn impossible_predicates<'tcx>(
```

Comment:
Wait, you never flipped the `bool` value. So currently it's still "predicates appear plausible", not "predicates are impossible"

Reply:
Fixed!